themes-default/slim/views/home.mako

```diff
@@ -21,6 +21,100 @@ const startVue = () => {
             return {
                 header: 'Show List'
             };
+        },
+        mounted() {
+            $('#showListTableSeries:has(tbody tr), #showListTableAnime:has(tbody tr)').tablesorter({
+                debug: false,
+                sortList: [[7, 1], [2, 0]],
+                textExtraction: (function() {
+                    return {
+                        0: node => $(node).find('span').attr('timestamp'),
+                        1: node => $(node).find('span').attr('timestamp'),
```

Comment:
I got it working by changing these two to:
```js
                        0: node => $(node).find('span').attr('timestamp'),
                        1: node => $(node).find('span').attr('timestamp'),
```

Reply:
Oh... I forgot I had thought about using a span with timestamp but then went with the old format.